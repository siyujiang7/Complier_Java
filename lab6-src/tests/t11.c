void main()
{
  long x, z;
  char* y;
  x = 2 * (3 + 4);
  z = 10 / 3;
  y = "world";
  printf("x: %ld, y: %s, imm: %ld, z: %ld\n", x, y, 5, z);
}
