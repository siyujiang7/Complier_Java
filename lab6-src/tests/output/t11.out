x: 14, y: world, imm: 5, z: 3
